elasticsearch/elasticsearch.yml

```diff
@@ -1,19 +1,21 @@
 ---
-  cluster.name: "AI-Hunter-Docker-Single-Node"
-  network.host: 0.0.0.0
+cluster.name: "BeaKer-ElasticSearch"
```

Comment:
Nit-pick note: Elasticsearch is styled on elastic.co/elasticsearch with a lowercase 's' at the start of search